hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/catalyst/catalog/HoodieCatalogTable.scala

```diff
@@ -335,6 +335,11 @@ class HoodieCatalogTable(val spark: SparkSession, var table: CatalogTable) exten
       }.partition(f => partitionFields.contains(f.name))
       StructType(dataFields ++ partFields)
     })
+    catch {
+      case cause: Throwable =>
+        logWarning("Failed to load table schema from meta client.", cause)
```

Comment:
Should we ignore all the exceptions?

Reply:
Since this function returns `Option` value, and we log the details of exception here, so I think it's OK to handle for all exceptions.